source/api/app.py

```diff
@@ -351,10 +351,10 @@ def get_data_columns():
 
 def log_request_parameters():
     logger.info("Processing the following request:\n")
-    logger.info("resource path: " + app.current_request.context['resourcePath'])
-    logger.info("method: " + app.current_request.method)
-    logger.info("uri parameters: " + str(app.current_request.uri_params))
-    logger.info("query parameters: " + str(app.current_request.query_params))
-    logger.info("request ID: " + app.current_request.context.get('requestId'))
-    logger.info('request body: ' + app.current_request.raw_body.decode())
+    logger.info(f"resource path: {app.current_request.context['resourcePath']}")
```

Comment:
Please don't reformat the print statement like this unless there is a defect.

Reply:
Thank you for trying to make this code follow the latest python standards. I recommend creating a new issue to track what you are trying to achieve, because there are other places where we concat strings with `+`. Be sure to include your desired outcome and an answer to *why* (e.g. make code more readable, make it faster, etc) so that we can justify spending time on it.

Meanwhile, please don't reformat the print statements here so that we can keep our PR and issue tracking as organized as possible.

Reply:
ok, i have revert to the old way.